### PR TITLE
Add some documentation around configuration and setup

### PR DESCRIPTION
**Pull request by robsiemb**
Warning: This renames several variables and will thus break deployments.  In particular:

    ZULIP_PUBLIC -> PUBLIC_TWO_WAY_STREAM
    ZULIP_STREAM -> ZULIP_LOG_PUBLIC_STREAM
    ZULIP_PRIVATE -> ZULIP_LOG_PRIVATE_STREAM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Zulip / Slack / Groupe Bridge Bot
```
Comment (timparenti) [inline README.md]: Groupme

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+1. Install redis.  A simple way to do this if you have docker installed is:
+
+```
+docker run -d --rm -p 6379:6379 --name redis_master redis:5.0
+```
+
+2. Create a slack bot account with bot user, add that bot to the appropriate channel on a slack.
+3. Create a zulip bot account on the zulip instance you intend to run.
+4. If you are using groupme, get a groupe bot account.
```
Comment (timparenti) [inline README.md]: groupme

File: secrets.py [hunk 3]
```diff
@@ -1,23 +1,34 @@
...
 # Please fill in all constants before operating
-SLACK_BOT_ID = 'U00000000'
-SLACK_ERR_CHANNEL = 'U00000000'
-ZULIP_STREAM = 'my stream'
-ZULIP_BOT_NAME = 'ack-bot'
-ZULIP_BOT_EMAIL = 'some-bot@site.zulipchat.com'
+
+# Authentication / Authorization configuration
+SLACK_BOT_ID = 'U00000000'                       # User ID of Bot
+ZULIP_BOT_NAME = 'some-bot'                      # Loclapart of bot email
```
Comment (timparenti) [inline secrets.py]: Localpart

**Comment by robsiemb** [comment]
Sorry about the typos!

**Comment by DaAwesomeP** [comment]
The force is strong in this one.

I'm going to assume this is tested and merge.

**Pull request merged**